src/main/java/uk/gov/ons/ctp/response/casesvc/CaseSvcApplication.java

```diff
@@ -231,4 +232,9 @@ public DistributedLockManager reportDistributedLockManager(RedissonClient rediss
         redissonClient,
         appConfig.getDataGrid().getReportLockTimeToLiveSeconds());
   }
+
+  @Bean
+  public Clock clock() {
```

Comment:
Not sure you need a singleton bean for such things . I Think you should just call it where you need it.